doc/classes/Light3D.xml

```diff
@@ -60,16 +60,16 @@
 			[Texture2D] projected by light. [member shadow_enabled] must be on for the projector to work. Light projectors make the light appear as if it is shining through a colored but transparent object, almost like light shining through stained glass.
 		</member>
 		<member name="light_size" type="float" setter="set_param" getter="get_param" default="0.0">
-			The size of the light in Godot units. Only available for [OmniLight3D]s and [SpotLight3D]s.
+			The size of the light in Godot units. Only available for [OmniLight3D]s and [SpotLight3D]s. Increasing this value will make the light fade out slower and shadows appear blurrier. This can be used to simulate area lights to an extent.
```

Comment:
```suggestion
			The size of the light in Godot units. Only available for [OmniLight3D]s and [SpotLight3D]s. Increasing this value will make the light fade out more slowly and shadows appear blurrier. This can be used to simulate area lights to an extent.
```

Reply:
Not 100% sure about this one, some native English speaker can likely confirm :)

Reply:
I applied it already, but I think "slower" was correct here.

Reply:
Apparently *slow* can be both adjective and adverb, so *fade out slower* should be correct as verb + comparative adverb. https://english.stackexchange.com/a/78248

Reply:
Alright, I changed it back.